test/load_transactions_use_case_tests.dart

```diff
@@ -0,0 +1,121 @@
+import 'dart:async';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:simple_result/simple_result.dart';
+import 'package:transactions/Feature/LocalTransactionLoader.dart';
+import 'package:transactions/Feature/Transaction.dart';
+import 'package:transactions/Store/TransactionStore.dart';
+import 'package:tuple/tuple.dart';
+
+void main() {
+  test("test_init_doesNotLoadAnyTransactions", () {
+    final tuple = _makeSUT();
+    final store = tuple.item2;
+
+    expect(store.messages.isEmpty, true);
+  });
+
+  test("test_load_requestLoadTransactions", () {
+    final tuple = _makeSUT();
+    final sut = tuple.item1;
+    final store = tuple.item2;
+
+    sut.load();
+
+    expect(store.messages, [TransactionStoreSpyMessage.load]);
+  });
+
+  test("test_loadTwice_requestLoadTransactionsTwice", () {
+    final tuple = _makeSUT();
+    final sut = tuple.item1;
+    final store = tuple.item2;
+
+    sut.load();
+    sut.load();
+
+    expect(store.messages, [TransactionStoreSpyMessage.load, TransactionStoreSpyMessage.load]);
+  });
+
+  test("test_load_deliversErrorOnLoadTransactionsError", () async {
+    final expectedError = TransactionStoreUnknownException();
+    final store = TransactionStoreStub(Result.failure(expectedError));
+    final sut = LocalTransactionLoader(store);
+
+    final result = sut.load();
+    result
+        .then((value) => () {
+              assert(false, "Expect failure, got : " + value.toString());
```

Comment:
This LoC is working as intended right? (It is exist within anonymous function)

```
assert(false, "Expect failure, got : " + value.toString());
```

Reply:
yes, all the tests are green ✅